act-rules-format.md

```diff
@@ -206,6 +213,71 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Mapping Secondary Requirements ####
+An accessibility requirement is mapped as a secondary requirement in a rule when both of the following are true: 
+
+- (a.) When one or more of the outcomes for a test target is `failed`, the accessibility requirement is <dfn>not satisfied</dfn> or <dfn>further testing is needed</dfn> for the test subject, or
+- (b.) When all of the outcomes are `passed` or `inapplicable`, the accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn> or <dfn>not satisfied</dfn>.
+
```

Comment:
the (a.) and (b.) in lines 219 and 220 are temporary markers. They are referenced in the temporary notes in lines 224 - 226.